### DISCUSSION TITLE
MyAAC v0.8.18

### DISCUSSION THREAD

**Discussion by slawkens** [slawkens/myaac#256] [Announcements]
### Added
* hook in guilds page to support guild wars (https://github.com/slawkens/myaac/commit/f875f3cd2059fac5c23a08ce73dd8621a66613e0)

<hr /><em>This discussion was created from the release <a href='https://github.com/slawkens/myaac/releases/tag/v0.8.18'>MyAAC v0.8.18</a>.</em>

**Comment by l3k0t** [comment]
god nice php 7.2 working
